src/inject/dynamic-theme/css-rules.ts

```diff
@@ -17,13 +18,8 @@ export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule
             } catch (err) {
                 if (err.message === "Cannot read property 'cssRules' of null") {
                     logInfo('An possible attempt on a unloaded CSSImportRule has been made.');
-                        (rule.parentStyleSheet.ownerNode as HTMLLinkElement).onload = () => {
-                        try {
-                            iterateCSSRules(rule.styleSheet.cssRules, iterate);
-                        } catch (err) {
-                            logWarn(err);
-                        }
-                    };
+                    await linkLoading(rule.parentStyleSheet.ownerNode as HTMLLinkElement);
+                    iterateCSSRules(rule.styleSheet.cssRules, iterate);
```

Comment:
thanks

Reply:
It's still a POC as it doesn't work for me so not sure where the fault relies.